### PR TITLE
Fix iojs compatibility by using fsevents 0.3.8

### PR DESCRIPTION
**Pull request by gravityrail**
This makes things work with iojs

### CODE REVIEW

**Comment by saper** [comment]
I am afraid this will not work - see https://github.com/sass/node-sass/issues/1107 and https://github.com/strongloop/fsevents/issues/82. Latest node-sass and fsevents do not seem to get along together.

**Comment by xzyfer** [comment]
Thanks @gravityrail but as per https://github.com/sass/node-sass/issues/1107#issuecomment-134867296 `fsevents@0.3.8` introduces a segfault which breaks Node Sass. Locking to `fsevents@0.3.7` as an attempt to prevent npm install the broken the version https://github.com/sass/node-sass/pull/1108.

Yes. Upgrading to `fsevents@0.3.8` will give you iojs v3 compatibility. However it'll mean you cannot use Node Sass.

**TLDR** don't use iojs v3 for anything that matters yet.
